code/datums/station_traits/negative_traits.dm

```diff
@@ -134,3 +134,85 @@
 		/// The bot's language holder - so we can randomize and change their language
 		var/datum/language_holder/bot_languages = found_bot.get_language_holder()
 		bot_languages.selected_language = bot_languages.get_random_spoken_language()
+
+/datum/station_trait/revenge_of_pun_pun
+	name = "Revenge of Pun Pun"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	var/static/list/weapon_types
+
+/datum/station_trait/revenge_of_pun_pun/New()
+	if(!weapon_types)
+		weapon_types = list(
+			/obj/item/chair = 20,
+			/obj/item/tailclub = 10,
+			/obj/item/melee/chainofcommand/tailwhip = 10,
+			/obj/item/melee/chainofcommand/tailwhip/kitty = 10,
+			/obj/item/reagent_containers/food/drinks/bottle = 20,
+			/obj/item/reagent_containers/food/drinks/bottle/kong = 5,
+			/obj/item/switchblade = 10,
+			/obj/item/sign/random = 10,
```

Comment:
`/obj/item/sign/random` doesn't seem to exist in the code anywhere

Reply:
It's *in this pr*